### PR TITLE
Check CLA from service

### PR DESCRIPTION
**Pull request by beliaev-maksim**
check users from service instead of GH org

### CODE REVIEW

**Review by seb128 — APPROVED** [review]
Thanks!

**Comment by townsend2010** [comment]
I'm seeing this error now: https://github.com/canonical/multipass/actions/runs/8666117384/job/23766251856?pr=1946#step:2:54

**Comment by beliaev-maksim** [comment]
@townsend2010 yes, also saw it in another repo

here is the fix, waiting to be merged
https://github.com/canonical/has-signed-canonical-cla/pull/41

File: index.js [hunk 1]
```diff
@@ -114,30 +111,30 @@ async function run() {
...
-      org: 'CanonicalContributorAgreement',
-      username: username
-    }).then((result) => {
-      if (result.status == 204) {
+    const axios = require('axios');
+
+    try {
+      console.log('Check in the signed list service');
+      const response = await axios.get(
+        'http://ec2-3-71-108-9.eu-central-1.compute.amazonaws.com/check_user/' + username
```
Comment (hpidcock) [inline index.js:119]: This should be over HTTPS and should be hosted on a canonical owned domain.

**Pull request merged**